src/stratis_cli/_parser/_parser.py

```diff
@@ -70,50 +107,22 @@ def gen_parser():
     :rtype: ArgumentParser
     """
     parser = argparse.ArgumentParser(
-       description="Stratis Storage Manager",
-       prog='stratis'
+        description="Stratis Storage Manager",
```

Comment:
This through line 121 breaks the indenting policy of this library...4 spaces if semantic indentation, 3 for non-semantic.